File: backend/components/metiers/threed/tests_centralises/legacy/samples/access/test_access_sample.js
// Test ultra avancé pour access_sample.js
const accessSample = require('../../../legacy/samples/access/access_sample');

describe('Access Sample', () => {
  it('doit valider l\'accès admin', () => {
    expect(accessSample('admin', 'secret')).toBe(true);
  });
  it('doit refuser un accès non admin', () => {
    expect(accessSample('user', 'wrong')).toBe(false);
  });
  // ...autres tests avancés, edge cases, sécurité...
});

File: backend/components/metiers/threed/tests_centralises/legacy/samples/access/test_access_sample.py
"""
Test ultra avancé pour access_sample.py
"""

from threed.legacy.samples.access.access_sample import access_sample


def test_access_admin():
    assert access_sample("admin", "secret") is True


def test_access_non_admin():
    assert access_sample("user", "wrong") is False


# ...autres tests avancés, edge cases, sécurité...
